app/client/reducers/transferAccountReducer.js

```diff
@@ -18,6 +19,9 @@ export const RESET_SELECTED = "RESET_SELECTED";
 const byId = (state = {}, action) => {
   switch (action.type) {
       case UPDATE_TRANSFER_ACCOUNTS:
+        if (action.reload) {
+          return action.transfer_accounts;
+        }
```

Comment:
return new object if reload (server side pagination) throws out all old objects.

this is needed so that react table handles data easily, without any big changes. also will still correctly handle pusher and other additions

Reply:
client now handling pagination